packages/core/src/components/inputs/SelectInput/SelectInput.vue

```diff
@@ -1,9 +1,9 @@
 <template>
   <div class="floating-input-group">
     <multiselect
-      :id="id"
+      :id="props.id"
       v-model="selectedOption"
-      :options="options"
+      :options="props.options"
```

Comment:
The props are uwrapped automatically inside the template so you do not need to use the `props.id` so just used `id`

Reply:
I think both works, but the ide just shows error/warning without that. But should I revert it back?

Reply:
Looks like a false positive from the IDE. I would prefer the version without the props prefix.